core/chains/evm/client/errors_test.go

```diff
@@ -127,6 +127,7 @@ func Test_Eth_Errors(t *testing.T) {
 			{"call failed: AlreadyKnown", true, "Nethermind"},
 			{"call failed: OwnNonceAlreadyUsed", true, "Nethermind"},
 			{"known transaction", true, "Klaytn"},
+			{"known transaction. transaction with hash 0x6013…3053 is already in the system", true, "zkSync"},
```

Comment:
Could you change this to the exact string we saw on the logs:
ErrorObject { code: ServerError(3), message: \"known transaction. transaction with hash 0x7a24…4a64 is already in the system\", data: Some(RawValue(\"0x\")) }

Also try to see how this test behaves with and without your changes in above file.

To verify this, without your changes, this test should fail. And after your above fix, this test should pass.